velox/docs/functions/spark/datetime.rst

```diff
@@ -9,4 +9,26 @@ These functions support TIMESTAMP and DATE input types.
 
 .. spark:function:: year(x) -> integer
 
-    Returns the year from ``x``.
\ No newline at end of file
+    Returns the year from ``x``.
+
+.. spark:function:: unix_timestamp() -> integer
+
+    Returns the current UNIX timestamp in seconds.
+
+.. spark:function:: unix_timestamp(string) -> integer
+
+    Returns the UNIX timestamp of time specified by ``string``. By default
+    assume the format ``yyyy-MM-dd HH:mm:ss``.
+
+.. spark:function:: unix_timestamp(string, format) -> integer
+
+    Returns the UNIX timestamp of time specified by ``string`` using the
+    format described in the ``format`` string.
+
+.. spark:function:: to_unix_timestamp(string) -> integer
```

Comment:
Are these part of SparkSQL or Meta-specific? If the latter, perhaps, these should go into Meta-specific repo.